code/modules/hydroponics/grown/root.dm

```diff
@@ -28,10 +28,18 @@
 /obj/item/food/grown/carrot/attackby(obj/item/I, mob/user, params)
 	if(I.get_sharpness())
 		to_chat(user, span_notice("You sharpen the carrot into a shiv with [I]."))
-		var/obj/item/knife/shiv/carrot/Shiv = new /obj/item/knife/shiv/carrot
+		var/Cblade
+		if(seed.potency > 25)
+			var/carrot_sword_chance = (seed.potency - 25) * 10
+			if (rand(1, 100) < carrot_sword_chance)
```

Comment:
The chances seem way too high for a baggable weapon comparable to an esword in power.
With this formula, a <35 potency carrot is guaranteed to be crafted into a sword.
Did you mean to divide `seed.potency - 25` by 10 rather than multiply it? That would make 100 potency carrots only have a 7.5% chance to be crafted into swords.

Reply:
made it weaker and less likely, better?

Reply:
I can't really say for certain. I think a maintainer is going to have to chime in on what numbers would be balanced.

Reply:
Personally for me I think that less likely would be far better. Making it a rare fun thing rather than a boring thing that always happens, yknow?

Reply:
made it a lot rarer. Thought 50 was the normal potency limit, my hubris.